lib/utils/isAutoprefixable.js

```diff
@@ -54,29 +303,24 @@ module.exports = {
 	 * @returns {boolean}
 	 */
 	property(identifier) {
-		return Boolean(autoprefixer.data.prefixes[prefixes.unprefixed(identifier.toLowerCase())]);
+		const ident = identifier.toLowerCase();
+
+		if (vendor.prefix(ident).length === 0) {
+			return false;
+		}
+
+		return PROPERTIES.has(vendor.unprefixed(ident));
```

Comment:
[note] This `property()` logic change requires the fix of `lib/rules/property-no-vendor-prefix/__tests__/index.js`.
Because `autoprefixer` can fix `-ms-interpolation-mode` to `image-rendering`, but this new simple implementation cannot support such a difficult case. I think the limitation is acceptable.

See also <https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering#examples>

Reply:
I reconsidered. I think it acceptable to just adding a `-ms-interpolation-mode` hack (bd874652).